.circleci/config.yml

```diff
@@ -88,14 +93,15 @@ jobs:
           at: ~/repo/
       - restore_cache:
           keys:
-            - v3-dependencies-{{ checksum "Pipfile.lock" }}
-      - run: *install-pipenv
-      - run: pip install aws-sam-cli
+            - v4-dependencies-{{ checksum "uv.lock" }}
+      - install_uv
+      - run: uv venv
```

Comment:
There's a couple of places we're explicitly calling `uv venv`.
Are they actually doing anything?
There's one here

Reply:
I added uv venv because without it the [job failed](https://app.circleci.com/pipelines/github/DemocracyClub/ec-postcode-lookup-pages/498/workflows/ce0fbc9e-dbb7-4026-9434-d19ab0436e50/jobs/2193) with

Reply:
I added uv venv because without it the [job failed](https://app.circleci.com/pipelines/github/DemocracyClub/ec-postcode-lookup-pages/498/workflows/ce0fbc9e-dbb7-4026-9434-d19ab0436e50/jobs/2193) with 

> error: No virtual environment found; run `uv venv` to create an environment, or pass `--system` to install into a non-virtual environment

 So I added uv venv to fix it

Reply:
The reason we were having to run `uv venv` was because we weren't actually caching the venv. Fixing the cached paths in https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117/commits/7cab83a01f204b6852297e55d2bc3901ee232f14 fixes this